pf/internal/convert/convert.go

```diff
@@ -70,6 +70,7 @@ func DecodePropertyMap(dec Decoder, v tftypes.Value) (resource.PropertyMap, erro
 	if err != nil {
 		return nil, err
 	}
+
```

Comment:
Needless diff?

Reply:
I had a change there that turned out to be unnecessary. Fixed.